schema/dataKeys.json

```diff
@@ -0,0 +1,352 @@
+        },
+        "product_stream": {
+          "type": "string",
+          "description": "The product stream e.g. RHEL-tp1"
+        },
+        "cpe": {
+          "type": "string",
+          "description": "The product CPE ID e.g. cpe:/a:example:openstack:el8"
+        },
+        "type": {
+          "type": "string",
+          "enum": ["RHEA", "RHBA", "RHSA"],
+          "description": "Type advisory type e.g. RHSA"
+        },
+        "synopsis": {
+          "type": "string",
+          "description": "The advisory synopsis e.g. my advisory synopsis"
+        },
+        "topic": {
+          "type": "string",
+          "description": "The advisory topic e.g. my advisory topic"
+        },
+        "description": {
+          "type": "string",
+          "description": "The advisory description e.g. This advisory is for security fixes to my product"
+        },
+        "solution": {
+          "type": "string",
+          "description": "The advisory solution e.g. For details on how to apply this update, see docs.com"
+        },
+        "reference": {
+          "type": "array",
+          "description": "The advisory references e.g. https://access.redhat.com/security/updates/classification & https://docs.redhat.com/some/example/release-notes ",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "sign": {
+      "type": "object",
+      "properties": {
+        "cosignSecretName": {
+          "type": "string",
+          "description": "Name of secret which contains AWS_REGION, AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY and SIGN_KEY"
+        },
+        "request": {
+          "type": "string",
+          "description": "The signing pipeline name to handle the request"
+        },
+        "configMapName": {
+          "type": "string",
+          "description": "The configmap that exists on the cluster for signing of the images e.g. hacbs-signing-pipeline-config-redhatbeta2"
+        },
+        "pipelineImage": {
+          "type": "string",
+          "description": ""
+        }
+      }
+    },
+    "github": {
+      "type": "object",
+      "properties": {
+        "githubSecret": {
+          "type": "string",
+          "description": "The secret used to authenticate GitHub via the the GitHub token"
+        }
+      }
+    },
+    "pyxis": {
+      "type": "object",
+      "properties": {
+        "secret": {
+          "type": "string",
+          "description": "The secret used to authenticate Pyxis e.g. example-collect-pyxis-params-cert"
+        },
+        "server": {
+          "enum": [
+            "stage",
+            "production",
+            "production-internal",
+            "stage-internal",
+            "none"
```

Comment:
This doesn't seem right. Should probably be removed?

Reply:
I have removed it .. fixed